database-sample-data.json

```diff
@@ -138,9 +140,18 @@
     },
     "questions": {
       "question_id_1": {
-        "name": "Deploy Sardaukarů?"
+        "name": "Deploy Sardaukarů?",
+        "questionType": "mandat_sardaukaru",
+        "byDelegateId": "delegate_id_1",
+        "byVotingRight": "Harko",
+        "roundId": "round_id_1",
+        "hidden": false
       }
     },
+    "questionsConfig": {
```

Comment:
Imo filters don't have to be persisted in db, it's normal to keep it only in the component. But if you already have it, don't remove it.

Reply:
Ok that's true, I kinda thought that if Ertai want's to come back to the same state as he left it, then it is nicer to have it in the DB. But for the next time then.